app/__tests__/assert.test.ts

```diff
@@ -8,7 +8,19 @@ interface User {
 
 define('Test Assertions', async () => {
   // Test 1: Basic Assertions (Will Pass)
+  afterAll(async () => {
+    console.log('Clearing DB after all tests');
+  });
+  beforeAll(async () => {
+    console.log('Clearing DB before all tests');
+  });
    new UITestBuilder<User>('/')
```

Comment:
Would be great to remove this, it's quite confusing to a user what new UITestBuilder is.

Reply:
The README shows what every test should look like:

```
import { define } from 'shortest'

define('Login Flow', () => {
  test('User can login')
    .given('I am on the login page')
    .when('I enter my credentials')
    .expect('I should be logged in')
})
```

The fewer lines of code and chars required the better, as it's in the name!

Reply:
IMO we could shorten further:

import { test } from 'shortest'

test('the user can login');

AI should be able to infer the rest.

Reply:
I absolutely agree. We should make it more intuitive and we can start by removing extra stuff. maybe we should just have test('', ()=>{}) and an optional callback function in case some operations need to happen in between. I will make this improvement.